legacy-tests/fixtures/index.js

```diff
@@ -32,11 +32,10 @@ describe('custom resolver', () => {
           customResolver: this.customResolver,
           nameFormatter: (path) => parse(path).name,
         },
-        storage: 'sequelize',
-        storageOptions: {
+        storage: new SequelizeStorage({
           path: this.storagePath,
-          sequelize: this.sequelize,
-        },
+            sequelize: this.sequelize,
+          }),
```

Comment:
The formatting lint rule (prettier) is turned off for `legacy-tests`, to avoid noise (there were a _lot_ of violations). I have a branch where I move most of these legacy tests to jest, which formats them as a side-effect. Will fix this one manually though.

Reply:
Ah, indeed, I believe I actually suggested turning it off for legacy-tests myself :sweat_smile: ok!!